Pipelines/Templates/deploy-Solution.yml

```diff
@@ -295,4 +295,34 @@ steps:
     makerPortalUrl: '$(TestAutomationMakerPortalUrl)'
     loginMethod: '$(TestAutomationLoginMethod)'
 
-- template: Hooks\deploy-solution-post-hook.yml
+# PowerPages folder contain only one subfolder which is WebSite
+- pwsh: |
+    $websiteName = ""
+    $solutionUnpackedFolder = "$(Build.SourcesDirectory)\$(Build.Repository.Name)\${{parameters.solutionName}}\PowerPages"
+    Write-Host "solutionUnpackedFolder - $solutionUnpackedFolder"
+    if(Test-Path "$solutionUnpackedFolder")
+    {
+        
+        $matchedFolders = Get-ChildItem $solutionUnpackedFolder -Directory | select Name
+
+        if($matchedFolders){
+          $websiteName = $matchedFolders[0].Name
+        }
+        echo "##vso[task.setvariable variable=WebsiteName]$websiteName"
+    }
+    else
+    {
+       Write-Host "Invalid path - $solutionUnpackedFolder"
```

Comment:
"Invalid Path" verbiage change. Add comment that "Single website is being processed"